easybuild/easyblocks/generic/pythonpackage.py

```diff
@@ -259,6 +265,8 @@ def extra_options(extra_vars=None):
                                            "to be the requirements file.", CUSTOM],
             'use_setup_py_develop': [False, "Install using '%s' (deprecated)" % SETUP_PY_DEVELOP_CMD, CUSTOM],
             'zipped_egg': [False, "Install as a zipped eggs (requires use_easy_install)", CUSTOM],
+            'postextinstallcmds': ['', "List of additional commands to execute "
```

Comment:
```suggestion
            'postextinstallcmds': [[], "List of additional commands to execute "
```
This is a list to I think this needs to initialised as one